examples/vector_databases/redis/redis-hybrid-query-examples.ipynb

```diff
@@ -0,0 +1,862 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cb1537e6",
+   "metadata": {},
+   "source": [
+    "# Running Hybrid VSS Queries with Redis and OpenAI\n",
+    "\n",
+    "This notebook provides an introduction to using Redis as a vector database with OpenAI embeddings and running hybrid queries that combine VSS and lexical search using Redis Search and Query capability. Redis is a scalable, real-time database that can be used as a vector database when using the [RediSearch Module](https://oss.redislabs.com/redisearch/). The RediSearch module allows you to index and search for vectors in Redis. This notebook will show you how to use the RediSearch module to index and search for vectors created by using the OpenAI API and stored in Redis.\n",
+    "\n",
+    "### What is Redis?\n",
```

Comment:
@michaelskyuan I like the way you frame and describe redis, redisearch, the modules, and deployment options...

Though, instead of putting this here, can you commit an update to the main redis examples README that gently merges in some of this content with what is existing?

Reply:
For this notebook, the intro can be much shorter / concise. The first paragraph is solid, but need to clarify that this notebook will show you how to index + search for vectors etc etc but also showcase hybrid search functionality in redis.

A short paragraph that explains hybrid filtering + search would also be appropriate.